Makefile

```
@@ -10,3 +10,10 @@ splash:
 
 icons:
 	flutter pub run flutter_launcher_icons
+
+
+.PHONY: build runner
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)